### ISSUE TITLE
EXE not proxy aware

### ISSUE THREAD

**Issue by mtbike61384** [bdamele/icmpsh#7]
Is there a flag for the slave .exe to point it to a proxy? Right now the host is behind a proxy and can't get back out of the network with ICMP.

**Comment by Pernat1y** [comment]
ICMP does not work behind proxy (if you talking about HTTP/Socks).

**Comment by mtbike61384** [comment]
Yeah thanks. I realized ICMP cannot work over HTTP after some more research so thanks.

**Comment by PowerLiuuu** [comment]
![Uploading 3c4665322917b3d87bb17723c85d2ef.png…]()
